src/templates/nav.html

```diff
@@ -31,7 +31,7 @@
   {# Desktop Nav Menu #}
   <div class="global-nav-menu-desktop">
     <h1 class="site">
-      <a href="{{ url('homepage') }}" class="mkt-wordmark"><span>Firefox Marketplace</span></a>
+      <span class="mkt-wordmark"><span>Firefox Marketplace</span></span>
```

Comment:
I would argue on desktop it should always take you back to the homepage since that's a time-tested design pattern.